google/resource_google_project_iam_policy_test.go

```diff
@@ -231,6 +231,12 @@ func TestAccGoogleProjectIamPolicy_basic(t *testing.T) {
 		Steps: []resource.TestStep{
 			// Create a new project
 			resource.TestStep{
+				Config: testAccGoogleProjectDefaultAssociatePolicyBasic(),
```

Comment:
Can you create a separate test for this called `TestAccGoogleProjectIamPolicy_defaultProject`

Reply:
will do, thanks